content/sdk/10.js/00.ethers/20.v6/00.getting-started.md

```diff
@@ -1,69 +1,134 @@
 ---
 title: "Getting Started"
 description: "Explore zkSync-ethers SDK version 6"
-tags: ["zksync", "sdks", "zkSync Era SDK", "ethereum", "era network", "javascript", "v6"]
+tags: ["zksync", "sdks", "zkSync Era SDK", "ethereum", "era network", "javascript", v6"]
 ---
 
-While most of the existing SDKs should work out of the box, deploying smart contracts or using unique zkSync features,
-like account abstraction, requires providing additional fields to those that Ethereum transactions have by default.
+While most existing SDKs work out of the box, deploying smart contracts or using unique zkSync features like account
+abstraction requires additional fields beyond the default Ethereum transaction fields. Understanding the types of objects
+and their responsibilities at a high level is useful:
 
-To begin, it is useful to have a basic understanding of the types of objects available and what they are
-responsible for, at a high level:
+- **Provider**: Connects to the zkSync Era blockchain, allowing querying of account, block, or transaction details, event
+logs, or read-only code evaluation using `call`. It also facilitates writing to the blockchain by sending transactions.
+- **Wallet**: Manages all operations interacting with an account, which typically has a private key for signing various
+payloads. It simplifies the use of common features.
 
-- `Provider` provides connection to the zkSync Era blockchain, which allows querying the blockchain state,
-  such as account, block or transaction details, querying event logs or evaluating read-only code using call.
-  Additionally, the client facilitates writing to the blockchain by sending transactions.
-- `Wallet` wraps all operations that interact with an account. An account generally has a private key, which
-  can be used to sign a variety of types of payloads. It provides easy usage of the most common features.
+::callout{icon="i-heroicons-light-bulb"}
+This example uses zkSync JavaScript SDK to interact with the zkSync Era blockchain.
+::
 
-## Examples
+- Create the scripts folder inside your project directory:
```

Comment:
We need to re-work this section. It's too confusing with the blocks of code broken up. As I'm reading through the guide, I don't know where the code is in relation to the other code for the zksync.ts file. Take a look at other examples for Getting Started: 
- https://jestjs.io/docs/getting-started
- https://docs.ethers.org/v5/getting-started/
- https://viem.sh/docs/getting-started

Reply:
I tried including steps just like this: https://viem.sh/docs/getting-started
You specified that the steps shouldn't be added that's why I wrote it without the steps